src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformedArr = [];
```

Comment:
what about this naming
```suggestion
  const stateHistory = [];
```